### DISCUSSION TITLE
ZEPETO

### DISCUSSION THREAD

**Discussion by Nena1739** [naverz/zepeto-studio-global#3] [Q&A]
Hola buenas noches me gustaría ser una creadora de ropa en ZEPETO y ayudar ala comunidad de ZEPETO espero y me podeis ayudar 🥺🥺

**Comment by meicatzpt** [comment]
Hello meow! You may learn in youtube and such~

**Comment by Ryanprw** [comment]
how to export sc to zepeto studio how? I'm always detected

**Reply by meicatzpt** [reply]
may I ask what sc is?

**Comment by jhslee** [comment]
Hello @Nena1739! I am the Zepeto Global Tech Evangelist! Its a pleasure to meet you! I'm so excited to hear you're interested in becoming a clothing creator for Zepetor!! 

Please refer to the following guide for information regarding our 3D item creation process~

https://studio.zepeto.me/guides/creating-your-item-3d-guide

The basic tools necessary to begin creating items are as follows: 

   -  Unity (Download)
        - Version: Unity 2020.3.9 (Required)
        - It must be installed to create ZEPETO items.
   -  ZEPETO Studio Unity Project files folder 3.1.11.1 (Download) - 2022.01.27. updated
        - Needed when converting to prefab file
        - Needed to convert prefab file to .zepeto extension file
   -  3D modeling tools such as Maya, 3ds Max, AutoCAD 3D
        - ZEPETO recommends using Maya. ZEPETO Studio's guide is based on Maya.
    - Image editing tools such as Adobe Photoshop
        - Needed to modify texture files

====================================================

¡Hola @Nena1739! ¡Soy el evangelista tecnológico global de Zepeto! ¡Es un placer conocerte! ¡Estoy tan emocionado de saber que estás interesado en convertirte en un creador de ropa para Zepetor!

Consulte la siguiente guía para obtener información sobre nuestro proceso de creación de artículos en 3D ~

https://studio.zepeto.me/guides/creating-your-item-3d-guide

Las herramientas básicas necesarias para comenzar a crear elementos son las siguientes:

    - Unity (Descargar)
         - Versión: Unity 2020.3.9 (obligatorio)
         - Debe estar instalado para crear artículos ZEPETO.
    - Carpeta de archivos ZEPETO Studio Unity Project 3.1.11.1 (Descargar) - 2022.01.27. actualizado
         - Necesario al convertir a un archivo prefabricado
         - Necesario para convertir un archivo prefabricado a un archivo con extensión .zepeto
    - Herramientas de modelado 3D como Maya, 3ds Max, AutoCAD 3D
         - ZEPETO recomienda utilizar Maya. La guía de ZEPETO Studio está basada en Maya.
     - Herramientas de edición de imágenes como Adobe Photoshop
         - Necesario para modificar archivos de textura
